subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy

```diff
@@ -59,12 +55,10 @@ class AbstractCrossVersionGradleProfilerPerformanceTest extends Specification {
 
     def setup() {
         def gradleProfilerReporter = new GradleProfilerReporter(temporaryFolder.testDirectory)
-        def slackReporter = reporter
-        def compositeReporter = CompositeDataReporter.of(slackReporter, gradleProfilerReporter)
         runner = new GradleProfilerCrossVersionPerformanceTestRunner(
             new GradleProfilerBuildExperimentRunner(gradleProfilerReporter.getResultCollector()),
             resultStore,
-            compositeReporter,
+            resultStore,
```

Comment:
Two identical params?

Reply:
We pass the same object once as a store once as a reporter.